src/main/views/crown-daily-list.njk

```diff
@@ -35,7 +35,6 @@
             <li>{{ restrictionBulletPoint1 }}</li>
             <li>{{ restrictionBulletPoint2 }}</li>
           </ul>
-          <p class="govuk-body">{{ restrictionInformationP5 }} <a href="#" class="govuk-link">{{ restrictionLink }}</a> </p>
```

Comment:
All the unused values like "restrictionInformationP5" and "restrictionInformationP5Bill" can be removed from relevant language files

Reply:
Hi Kian, removed restrictionInformationP5/restrictionInformationP5Bill and the associated restrictionLinks